javascript/index.js

```diff
@@ -14,52 +14,63 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  setInterval(()=>{
+  printMinutes(chronometer.twoDigitsNumber(chronometer.getMinutes()))
```

Comment:
This work but definitly not the best solution. The idea is to pass the callback that will run in the setInterval of the chronometer class. To do that you just want to pass the printTime function as a parameter inside the startClick call in line 60. and then Inside the startClick declaration just call the callback function.
Something like:
index.js Line 60: chronometer.stopClick(printTime) // the function is being passed to the chronometer object.
chronometer.js Line 9: callback() // the printTime function is being invoked inside the setInterval